'__builtin_ctzll': identifier not found while building Calculator example

I tried to run the calculator example and I run into this issue

```
$ react-native run-windows
 √ Auto-linking...
Success: No auto-linking changes necessary. (64ms)
 √ Found MSBuild v16.0 at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin (16.8.30717.126)
 √ Restoring NuGet packages
 √ Found Solution: D:\DEV\react-native-windows-samples\samples\Calculator\windows\Calculator.sln
 i Build configuration: Debug
 i Build platform: x86
 × Building Solution: C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\XamlCompiler\Microsoft.Windows.UI.X...
 × Build failed with message      
5>D:\DEV\react-native-windows-samples\samples\Calculator\node_modules\.folly\folly-2019.09.30.00\folly\portability\Builtins.h(109,15): error C3861:
'__builtin_ctzll': identifier not found (compiling source file D:\DEV\react-native-windows-samples\samples\Calculator\node_modules\.folly\folly-2019.09.30.00\folly\container\detail\F14Table.cpp) [D:\DEV\react-native-windows-samples\samples\Calculator\node_modules\react-native-windows\Folly\Folly.vcxproj].
Check your build configuration.
```

Also running `yarn start` does nothing. I see the react-native message on the console and waiting for a window to appear but nothing comes up. Pressing `r` or `d` logs a warning that no apps are connected.

Any help would be appreciated.

--- a/samples/Calculator/package.json
+++ b/samples/Calculator/package.json
@@ -12,7 +12,7 @@
   "dependencies": {
     "react": "16.13.1",
     "react-native": "0.63.2",
-    "react-native-windows": "^0.63.0-0"
+    "react-native-windows": "^0.63.10"
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",

